pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -132,6 +124,22 @@ func changefeedPlanHook(
 		ctx, span := tracing.ChildSpan(ctx, stmt.StatementTag())
 		defer span.Finish()
 
+		if err := featureflag.CheckEnabled(
+			ctx,
+			p.ExecCfg(),
+			featureChangefeedEnabled,
+			"CHANGEFEED",
+		); err != nil {
+			return err
+		}
```

Comment:
I'm moving this down into the returned `PlanHookRowFn` based on this discussion: https://cockroachlabs.slack.com/archives/C9TGBJB44/p1631200940138700